secrets-provider-chart/templates/secrets-provider.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: {{ include "conjur.deploy-api" . }}
+kind: Job
+metadata:
+  name: {{ .Release.Name }}
+  namespace: {{ .Release.namespace }}
+spec:
+  template:
+    spec:
+      serviceAccountName: {{ .Values.rbac.serviceAccount.name }}
+      containers:
+      - image: {{ .Values.secretsProvider.image }}:{{ .Values.secretsProvider.tag }}
+        imagePullPolicy: IfNotPresent
+        name: {{ .Values.secretsProvider.name }}
+        env:
+        - name: SECRETS_DESTINATION
+          value: k8s_secrets
+
+        - name: CONTAINER_MODE
+          value: init
+
+        - name: MY_POD_NAME
+          valueFrom:
+            fieldRef:
+              apiVersion: v1
+              fieldPath: metadata.name
+
+        - name: MY_POD_NAMESPACE
+          valueFrom:
+            fieldRef:
+              apiVersion: v1
+              fieldPath: metadata.namespace
+
+        - name: CONJUR_APPLIANCE_URL
+          value:  {{ .Values.environment.conjur.applianceUrl | quote }}
+
+        - name: CONJUR_AUTHN_URL
+          value:  {{ .Values.environment.conjur.authnUrl | quote }}
+
+        - name: CONJUR_ACCOUNT
+          value:  {{ .Values.environment.conjur.account | quote }}
+
+        - name: CONJUR_SSL_CERTIFICATE
+          valueFrom:
+            configMapKeyRef:
+              name: {{ .Values.environment.conjur.sslCertificate.name | quote }}
+              key: ssl-certificate
+
+        - name: CONJUR_AUTHN_LOGIN
+          value:  {{ .Values.environment.conjur.authnLogin | quote }}
+
+        - name: SUPPORT_MULTIPLE_APPS
```

Comment:
what's the purpose of this var?

Reply:
this was supposed to be a distinguishable of the job much like we have for `container_mode = init` so I believe so.

@abrahamko what did we decide on this?

Reply:
We need to add a var to distinguish Rotation too so we can already implement it in the code. @abrahamko 
For example: `SUPPORT_ROTATION`

Reply:
i don't understand what we will do differently when `SUPPORT_MULTIPLE_APPS` is true and when it's false. can you elaborate please?

Reply:
SUPPORT_MULTIPLE_APPS=true is the separately deployment Job that supports multiple apps
Ideally we wouldn't be expecting this value to be false for now because we have CONTAINER_MODE init for that

Reply:
@sigalsax it is not clear to me for what we need this variable. Please, explain what is the logic that it adds

Reply:
For init deployment we have CONTAINER_MODE=init, now we cannot use that anymore because we are no longer init container but we still need to identity the type of deployment so we thought `SUPPORT_MULTIPLE_APPS` is most representative of deployment _flow_

Reply:
so it is going to be instead of the variable CONTAINER_MODE ?

Reply:
we aren't deleting container_mode for backwards compatibility